src/Toppokki.js

```diff
@@ -0,0 +1,255 @@
+var path = require("path");
+var puppeteer = require("puppeteer");
+var browserify = null;
+var UglifyJS = null;
+
+try {
+  browserify = require('browserify');
```

Comment:
i.e. this and uglify should just be moved to a separate module so that if imported, they are used, but otherwise it does not affect the rest of the module